strawberry_django_jwt/middleware.py

```diff
@@ -110,8 +111,17 @@ def resolve(self, _next, root, info: GraphQLResolveInfo, *args, **kwargs):
         return _next(root, info, **kwargs)
 
 
+def channels_compat(context: StrawberryChannelsContext) -> None:
+    request = context.request
+    request.META = request.headers
+    request.COOKIES = request.scope["session"]
+
+
 class AsyncJSONWebTokenMiddleware(BaseJSONWebTokenMiddleware):
     async def resolve(self, _next, root, info: GraphQLResolveInfo, *args, **kwargs):
+        if isinstance(info.context, StrawberryChannelsContext):
```

Comment:
I think that is the first entry point as far as our concerns am I right? Also this is not in `BaseJSONWebTokenMiddleware` since channels won't use sync schema.